dlrover/python/master/node/job_context.py

```diff
@@ -48,15 +48,13 @@ def next_action(
     ):
         return self._action_queue.next_action(instance=instance)
 
-    @property
-    def ps_nodes(self) -> Dict[int, Node]:
+    def get_and_update_ps_nodes(self) -> Dict[int, Node]:
```

Comment:
There is no any code to update `job_nodes["PS"]`

Reply:
`get_and_update_ps_nodes` -> `get_mutable_ps_nodes`

Reply:
```get_mutable_ps_nodes ``` lgtm